lib/alephant/sequencer/sequence_table.rb

```diff
@@ -66,10 +66,24 @@ def update_sequence_id(ident, value, last_seen_check = nil)
             })
           end
 
-          logger.info("SequenceTable#update_sequence_id: with new value #{value} for #{ident} success!")
+          logger.info(
+            "event"  => "SequenceIdUpdated",
+            "id"     => ident,
+            "value"  => value,
+            "method" => "#{self.class}#update_sequence_id"
+          )
         rescue AWS::DynamoDB::Errors::ConditionalCheckFailedException
           logger.metric "SequencerFailedConditionalChecks"
-          logger.warn("SequenceTable#update_sequence_id: (Value to put: #{value}, existing: #{current_sequence}) #{ident} outdated!")
+          logger.error(
+            "event"                => "DynamoConditionalCheckFailed",
```

Comment:
@integralist Think it'll be better for future consistency to specify it as `DynamoDBConditionalCheckFailed`, as we're not using [Dynamo](https://en.wikipedia.org/wiki/Dynamo_(storage_system)) :)